oshdb-api/src/main/java/org/heigit/ohsome/oshdb/api/mapreducer/backend/MapReducerJdbcMultithread.java

```diff
@@ -20,10 +20,16 @@
 import org.jetbrains.annotations.NotNull;
 import org.json.simple.parser.ParseException;
 
-
+/**
+ * An implementation of the OSHDB API using a JDBC database as backend, where calculations run in
+ * parallel.
+ *
+ * <p>This implementation uses JAVA's {@link Stream#parallel()} implementation to run some
+ * operations concurrently.</p>
+ */
 public class MapReducerJdbcMultithread<X> extends MapReducerJdbc<X> {
-  public MapReducerJdbcMultithread(OSHDBDatabase oshdb,
-      Class<? extends OSHDBMapReducible> forClass) {
+  public MapReducerJdbcMultithread(
+      OSHDBDatabase oshdb, Class<? extends OSHDBMapReducible> forClass) {
```

Comment:
Was this change in the line break intended? If yes, what was wrong with the previous one?

Reply:
hm, not sure :see_no_evil: I will revert it back :+1: